terraform/uptime-checks/checks.tf

```diff
@@ -0,0 +1,81 @@
+# Setup local variables with list of hubs that we want checks for
+locals {
+  cluster_yamls = [for f in fileset(path.module, "../../config/clusters/*/cluster.yaml") : yamldecode(file(f))]
+  hubs = toset(flatten([for cy in local.cluster_yamls : [for h in cy["hubs"] : {
+    name       = h["name"],
+    domain     = h["domain"]
+    helm_chart = h["helm_chart"]
+    cluster    = cy["name"]
+    provider   = cy["provider"]
+  }]]))
+}
+
+resource "google_monitoring_uptime_check_config" "hub_simple_uptime_check" {
+  for_each = { for h in local.hubs : h.domain => h }
+
+  display_name = "${each.value.domain} on ${each.value.cluster}"
+  timeout      = "30s"
+
+  # Check every minute
+  period = "300s"
```

Comment:
This comment says "check every minute", but the period is 300 seconds (5 minutes) - which is it?

ETA: Judging by the comment on L61, the comment needs to be updated to "check every 5 mins"